catroid/src/main/java/org/catrobat/catroid/pocketmusic/ui/TactScrollRecyclerView.java

```diff
@@ -161,6 +179,9 @@ public int getItemViewType(int position) {
 
 		@Override
 		public int getItemCount() {
+			if (isPlaying) {
+				return Math.max(tactCount, MINIMUM_TACT_COUNT) + PLUS_BUTTON_ON_END + 1;
```

Comment:
... and would use the constant EMPTY_VIEW_COUNT_ON_END instead of PLUS_BUTTON_ON_END + 1